umm-spec-lib/src/cmr/umm_spec/models/umm_collection_models.clj

```diff
@@ -70,9 +70,9 @@
    ;; This element stores the DOI (Digital Object Identifier) that identifies the collection. Note:
    ;; The values should start with the directory indicator which in ESDIS' case is 10. If the DOI
    ;; was registered through ESDIS, the beginning of the string should be 10.5067. The DOI URL is
-   ;; not stored here; it should be stored as a RelatedURL. The DOI organization that is responsible
-   ;; for creating the DOI is described in the Authority element. For ESDIS records the value of
-   ;; http://dx.doi.org/ should be used.
+   ;; not stored here; it should be stored as a RelatedURL. Authority is the registry that registers
```

Comment:
Out of curiosity - how did you know the model needed to be updated? We should be catching this when we make the actual ticket change for things like this.

Reply:
🤔

Reply:
I don't. I just run the `generate-umm-records` task and this is changed as a result. 

I guess this is a result of the person who made the original schema change forgot to run the task to update the model before merging it to master. Since this only changed some comments, no tests failed. So everybody is happy.

Reply:
The humor ... it's so dry, I think I need skin lotion.